gen.go

```diff
@@ -24,4 +24,4 @@ package bindings
  * looked up in migration script here: https://github.com/mysteriumnetwork/payments-smart-contracts/blob/master/migrations/2_deploy_contracts.js
  */
 
-//go:generate go run bindings/abi/abigen.go --githubrepo=mysteriumnetwork/payments-smart-contracts -githubrelease=v0.4.5 --contracts=MystToken.json,MystDEX.json,ChannelImplementation.json,AccountantImplementation.json,Registry.json --out=bindings --pkg=bindings
+//go:generate go run bindings/abi/abigen.go --githubrepo=mysteriumnetwork/payments-smart-contracts -githubrelease=v0.4.5 --contracts=Migrations.json,SafeMathLib.json,MystToken.json,MystDEX.json,ChannelImplementation.json,AccountantImplementation.json,Registry.json --out=bindings --pkg=bindings
```

Comment:
Are you sure we need `Migrations.json`? This is just helper smart contract for truffle stuff.

Reply:
I'll retest and let you know